src/MigrationTools.Clients.TfsObjectModel/Tools/TfsEmbededImagesTool.cs

```diff
@@ -132,7 +132,15 @@ private string UploadedAndRetrieveAttachmentLinkUrl(string matchedSourceUri, str
                 {
                     if (!string.IsNullOrEmpty(sourcePersonalAccessToken))
                     {
-                        httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic", sourcePersonalAccessToken);
+                        if (!string.IsNullOrEmpty(sourcePersonalAccessToken) && matchedSourceUri.Contains("tfs/DefaultCollection"))
```

Comment:
This will only work for `tfs/DefaultCollection` and not for any other named collection in TFS.

maybe test the whole URL and see if its "dev.azure.com" or "*.tfsonline.com" and if not use the TFS auth?